osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -97,6 +97,7 @@ protected override void InitialiseDefaults()
             SetDefault(OsuSetting.MenuParallax, true);
 
             // Gameplay
+            SetDefault(OsuSetting.PositionalHitsoundsLevel, 0.8f, 0, 1);
```

Comment:
the default here changes behaviour. was that intentional? as far as i can tell, to preserve the previous balance it should be `0.2f` (because the value is doubled at usage site)